module/input/SensorFilter/src/SensorFilter.cpp

```diff
@@ -564,125 +571,133 @@ namespace module::input {
                                              MeasurementType::GYROSCOPE());
 
                         // Calculate accelerometer noise factor
-                        Eigen::Matrix3d acc_noise =
+                        const Eigen::Matrix3d acc_noise =
                             config.motionFilter.noise.measurement.accelerometer
+                            // Add noise which is proportional to the square of how much we are moving, minus gravity
+                            // This means that the faster we move, the noisier we think the measurements are
```

Comment:
"faster we move" is not really a phrase that matches with acceleration (large acceleration in negative direction could mean we are slowing down for instance)